### DISCUSSION TITLE
Fallback for non Cloudinary Links

### DISCUSSION THREAD

**Discussion by TonsiTT** [cloudinary-community/next-cloudinary#258] [Q&A]
Hi,
So I'm currently hosting images on a third-party provider and serving on a Next.js site. Now, my plan is to shift to Cloudinary using the [Lazy Migration Strategy](https://cloudinary.com/documentation/migration#lazy_migration_with_auto_upload). 
I have mapped the folder but there are sometimes links which are not mapped due to numerous reasons which are sort of out of hand.

So I was wondering if there is an option for a fallback option in case the link passed to the component is not a Cloudinary link and sort of use the normal img tag instead

**Comment by TonsiTT** [comment]
So I managed to write a function on my end to check if it is a URL that is mapped for lazy migration and then use _ClImage_  and otherwise to use standard image tag. 
However, I am unable to use SeoSuffix with _ClImage_.  I could appreciate some guidance in using that

**Reply by colbyfayock** [reply]
hey @TonsiTT you should be able to use the `seoSuffix` prop with CldImage. honestly i don't use it much in practice, but it should work along the lines of:

```
<CldImage
  src="my-image"
  ...
  seoSuffix="stars-above-a-mountain"
/>
```

**Reply by colbyfayock** [reply]
if you're seeing issues feel free to create a new Issue: https://github.com/colbyfayock/next-cloudinary/issues/new?assignees=&labels=Type%3A+Bug&projects=&template=bug_report.md&title=%5BBug%5D+
